gradle/libs.versions.toml

```diff
@@ -30,6 +30,7 @@ androidx-core = "androidx.core:core:1.17.0"
 androidx-annotation = "androidx.annotation:annotation:1.9.1"
 androidx-lifecycle-process = "androidx.lifecycle:lifecycle-process:2.9.2"
 androidx-preference-ktx = "androidx.preference:preference-ktx:1.2.1"
+apache-httpcore = "org.apache.httpcomponents:httpcore:4.4.16"
```

Comment:
Maybe it's worth adding a comment/todo to remove it once we no longer have Volley?